generator/build.gradle.kts

```diff
@@ -7,3 +10,41 @@ group = project.property("GROUP_ID") as String
 dependencies {
     api("org.openapitools:openapi-generator:7.11.0")
 }
+
+openApiGenerate {
+    inputSpec = System.getProperty("inputSpec") ?: "$projectDir/src/main/resources/specs.yaml"
+
+    packageName = "com.expediagroup.sdk.xap"
+    invokerPackage = "com.expediagroup.sdk.xap"
+    apiPackage = "com.expediagroup.sdk.xap.operations"
+    modelPackage = "com.expediagroup.sdk.xap.models"
+
+    dryRun = false
+    cleanupOutput = false
+    generateApiDocumentation = false
+    generateApiTests = false
+    generateModelTests = false
+    enablePostProcessFile = true
+
+    templateDir = "$projectDir/src/main/resources/templates"
+    configFile = "$projectDir/src/main/resources/generator-config.yaml"
+    outputDir = "$rootDir/xap-sdk/src/main/kotlin"
+
+    additionalProperties.put(CodegenConstants.ENUM_PROPERTY_NAMING, "UPPERCASE")
+    additionalProperties.put("allowedMediaTypes", AllowedMediaTypesLambda())
+
+    configOptions.put("sourceFolder", "")
+
+    globalProperties.put("supportingFiles", "Room.kt")
+}
+
+tasks.named("openApiGenerate").configure {
+    doLast {
+        exec {
```

Comment:
This is deprecated. What's replacing it?

Reply:
Updated.